pkg/api/serialization_test.go

```diff
@@ -388,8 +388,8 @@ func fuzzInternalObject(t *testing.T, forVersion schema.GroupVersion, item runti
 					}
 				}
 				if c.RandBool() {
-					params.MaxUnavailable = intstr.FromInt(int(c.RandUint64()))
-					params.MaxSurge = intstr.FromInt(int(c.RandUint64()))
+					params.MaxUnavailable = intstr.FromInt(int(c.RandUint64() >> 33))
```

Comment:
this is to make it 31 bits or less? surprised this broke fuzzing, I would only have expected a validation error